pkg/source/template.go

```diff
@@ -20,6 +21,18 @@ func (i InvalidPluginSpecError) Error() string {
 	return i.err
 }
 
+// for backward compatibility
+// by default addURIAndSha assumed 4 spaces indent
+func fixShaIndentation(v string) string {
```

Comment:
Would this fix if `addURISha` appeared on the opening `-`, like
```yaml
someList:
   - {{ addURISha "example.com" ... }
```

Reply:
yes, it will. it was not the problem with dash, but the indentation.